src/Sys/InspectSeoHandler.php

```diff
@@ -28,16 +28,20 @@ public static function handlesCuries(): array
     }
 
     public function __construct(
-        private readonly Ncr             $ncr,
-        private readonly Key             $key,
-        private readonly Flags           $flags,
-        private readonly LoggerInterface $logger = new NullLogger(),
+        private readonly Ncr                        $ncr,
+        private readonly Key                        $key,
+        private readonly Flags                      $flags,
+        private readonly LoggerInterface|NullLogger $logger = new NullLogger(),
```

Comment:
@digitalartcrew `NullLogger` implements `LoggerInterface` so this is redundant

> if class B extends A {}, then A|B is also a legal union type, even though it could be reduced to just A.
- https://www.php.net/manual/en/language.types.declarations.php#language.types.declarations.composite.redundant